src/database_metrics/README.md

```diff
@@ -0,0 +1,17 @@
+# Capturing Database Metrics
+This module contains methods for collecting metrics on the database server under different conditions.
+
+## Usage
+1. Using the [Docker tutorials](https://github.com/cmu-delphi/delphi-epidata/blob/main/docs/epidata_development.md),
```

Comment:
the backend guide might be more direct than the epidata guide (since the latter points to the former), but up to you

link to backend guide: https://github.com/cmu-delphi/operations/blob/main/docs/backend_development.md

Reply:
(nevermind, i see you use the web and database images below -- pls disregard)

Reply:
:+1: btw, those guides were super useful.

Reply:
thanks :D glad to hear that